src/main/java/core/basesyntax/BallsCollors.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum BallsCollors {

```

Comment:
Consider using the `name()` method to get the String representation of enum constants instead of `toString()` or `String.valueOf()`.